community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -78,27 +90,10 @@ This option has some additional requirements:
   development environment deploying the cluster. One can use following commands:
 
   ```bash
-  wget https://raw.githubusercontent.com/SchedMD/slurm-gcp/5.7.2/scripts/requirements.txt
-  pip3 install -r requirements.txt --user
+  pip3 install -r https://raw.githubusercontent.com/SchedMD/slurm-gcp/5.7.2/scripts/requirements.txt --user
```

Comment:
`--user` will throw if is run from virtualenv. Do we want to discourage usage of virtualenv?

Reply:
good point, also, it doesn't seem necessary if ran by non-root. It default to --user, right?

Reply:
I am reading that "pip defaults to installing Python packages to a system directory". I went through this process when testing and indeed the `--user` argument is denied when installing into virtualenv, but the error message is very clear. I would opt to leave it as we do not mention anything about a virtual environment. I also don't feel strongly so if you still disagree I can update.

Reply:
In fact I just ended up removing `--user` as it does not make much difference to me and I want to move forward with this PR without waiting for re-review.